liquidity_pool/src/liquidity.rs

```diff
@@ -234,75 +221,66 @@ pub trait LiquidityModule:
                 .direct(&initial_caller, asset_token_id, 0, &extra_asset_paid, &[]);
         }
 
-        // TODO: Instead of borrow_token_amount (i.e. 1:1 ratio), calculate how much collateral amount was repaid
```

Comment:
Keep this TODO, as it seems to me like the ratio is still 1:1

Reply:
done